opm/autodiff/FlowMainEbos.hpp

```diff
@@ -70,6 +71,8 @@ SET_STRING_PROP(EclFlowProblem, EnableDryRun, "auto");
 
 SET_INT_PROP(EclFlowProblem, OutputInterval, 1);
 
+SET_BOOL_PROP(EclFlowProblem, UseAdjoint, false);
+
```

Comment:
better call this `EnableAdjoint` (that's the standard convention for boolean properties in eWoms). also, the `NumAdjoints` property from the eWoms PR seems to be set neither there nor here. is it unused?!

Reply:
NumAdjoints is used to make larger AD variables for adjoint. EneableAdjoint used to deside if one is running adjoint simulation or not.